src/js/events/CosmosPackagesActions.js

```diff
@@ -149,6 +150,35 @@ const CosmosPackagesActions = {
     });
   },
 
+  fetchPackageVersions(packageName) {
+    // necessary for the backend
+    // even though it's not in use
+    const includePackageVersions = false;
+
+    RequestUtil.json({
+      contentType: getContentType("list-versions", "request", "v1"),
+      headers: { Accept: getContentType("list-versions", "response", "v1") },
+      method: "POST",
+      url: `${Config.rootUrl}${Config.cosmosAPIPrefix}/list-versions`,
+      data: JSON.stringify({ packageName, includePackageVersions }),
+      success(response) {
+        const packageVersions = response.results;
+
+        AppDispatcher.handleServerAction({
+          type: REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_SUCCESS,
+          data: packageVersions
+        });
+      },
+      error(xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_ERROR,
+          data: RequestUtil.getErrorFromXHR(xhr),
+          xhr
```

Comment:
You can also include `packageName` in the error object here.

Reply:
why though?

Reply:
I see it being done in the other tests. So I figure there is reason for it.

Reply:
yes I saw that too, I found no reason to be passing.